src/io/output_mesh.hh

```diff
@@ -49,9 +43,6 @@ class OutputMeshDiscontinuous;
 class OutputMeshBase : public std::enable_shared_from_this<OutputMeshBase>
```

Comment:
Describe work with this class. provide example of usage, how to call correct methods in correct order.